### DISCUSSION TITLE
pg_repack on managed postgresql

### DISCUSSION THREAD

**Discussion by andychongyz** [ubicloud/ubicloud#1022] [General]
Hi everyone! I just signed up for the GitHub runner and immediately saved 10 times on my bill. It's amazing! I have a question: Can I run pg_repack on managed PostgreSQL? Most managed solutions don't support it.

**Comment by byucesoy** [comment]
Right now, we only support contrib extensions, but we are planning to add more extensions soon. pg_repack could be one of them. Could you please explain your use-case a bit more? It would help us in our prioritization.

**Reply by andychongyz** [reply]
I use pg_repack to minimize locking while removing bloat. Here's an [article](https://pganalyze.com/blog/5mins-postgres-pg-repack-VACUUM-FULL) that compares the use of pg_repack and VACUUM FULL.

**Comment by enescakir** [comment]
Over the past few months, we've added several new extensions to our managed PostgreSQL service. `pg_repack` is one of them. You can see the list of available extensions for Ubicloud Managed PostgreSQL [here](https://www.ubicloud.com/docs/managed-postgresql/extensions)
